jsapp/js/components/map.es6

```diff
@@ -180,20 +192,46 @@ export class FormMap extends React.Component {
             iconSize: [20, 20],
           });
         }
-        prepPoints.push(L.marker(item._geolocation, {icon: icon, sId: item._id}));
+        prepPoints.push(
+          L.marker(item._geolocation, {
+            icon: icon, 
+            sId: item._id, 
+            typeId: viewby && mapMarkers ? mapMarkers[itemId].id : null
+          })
+        );
       }
     });
 
     if (prepPoints.length > 0) {
       if (viewby) {
         var markers = L.featureGroup(prepPoints);
       } else {
-        var markers = L.markerClusterGroup({maxClusterRadius: this.calculateClusterRadius, disableClusteringAtZoom: 16});
+        var markers = L.markerClusterGroup({
+          maxClusterRadius: this.calculateClusterRadius, 
+          disableClusteringAtZoom: 16,
+          iconCreateFunction: function(cluster) {
+            var childCount = cluster.getChildCount();
+
+            var c = ' marker-cluster-';
```

Comment:
Maybe it would be more obvious why space is here if you change it to `'marker-cluster marker-cluster-'`